src/endpoints/getTeamStats.ts

```diff
@@ -4,24 +4,75 @@ import { fetchPage, toArray, getTimestamp, getMapSlug } from '../utils/mappers'
 import { popSlashSource } from '../utils/parsing'
 
 export const getTeamStats = (config: HLTVConfig) => async ({
-  id
+  id,
+  currentRoster
 }: {
-  id: number
+  id: number,
+  currentRoster: boolean
 }): Promise<FullTeamStats> => {
-  const $ = await fetchPage(
+  let currentRosterURL = ''
+  let matchesURL = ''
+  let eventsURL = ''
+  let mapsURL = ''
+
+  let $ = await fetchPage(
     `${config.hltvUrl}/stats/teams/${id}/-`,
     config.loadPage
   )
+
+  const getContainerByText = (text) =>
+    $('.standard-headline')
+      .filter((_, el) => $(el).text() === text)
+      .parent()
+      .next()
+
+  const getPlayersByContainer = (container) =>
+    toArray(container.find('.image-and-label')).map((playerEl) => ({
+      id: Number(playerEl.attr('href')!.split('/')[3]),
+      name: playerEl.find('.text-ellipsis').text()
+    }))
+
+  const currentLineup = getPlayersByContainer(
+    getContainerByText('Current lineup')
+  )
+
+  const historicPlayers = getPlayersByContainer(
+    getContainerByText('Historic players')
+  )
+  const standins = getPlayersByContainer(getContainerByText('Standins'))
+
+  if (currentRoster) {
+    currentRosterURL = 'lineup=' + currentLineup[0].id
+    currentRosterURL = currentRosterURL + '&lineup=' + currentLineup[1].id
+    currentRosterURL = currentRosterURL + '&lineup=' + currentLineup[2].id
+    currentRosterURL = currentRosterURL + '&lineup=' + currentLineup[3].id
+    currentRosterURL = currentRosterURL + '&lineup=' + currentLineup[4].id
+    currentRosterURL = currentRosterURL + '&minLineupMatch=0'
```

Comment:
I feel like this can be refactored. Something like:
```js
currentRosterUrl = `lineup=${currentLineup.map(x => x.id).join('&lineup=')}&minLineupMatch=0`
```

Reply:
smart